packages/docusaurus/bin/docusaurus.js

```diff
@@ -312,8 +312,19 @@ function isInternalCommand(command) {
 }
 
 async function run() {
-  if (!isInternalCommand(process.argv.slice(2)[0])) {
-    await externalCommand(cli, path.resolve('.'));
+  const commandName = process.argv.slice(2)[0];
+
+  if (!isInternalCommand(commandName)) {
+    try {
+      await externalCommand(cli, path.resolve('.'));
+    } catch (err) {
+      console.log(
+        chalk.red(
+          `Running external '${commandName}' command failed.\n\n${err}`,
```

Comment:
I don't think users will understand the concept of "external command" so it's probably useless to mention it here?